compute_tools/src/rsyslog.rs

```diff
@@ -36,14 +53,14 @@ fn get_rsyslog_pid() -> Option<String> {
 // TODO: test it properly
 //
 fn restart_rsyslog() -> Result<()> {
-    let old_pid = get_rsyslog_pid().context("rsyslogd is not running")?;
-    info!("rsyslogd is running with pid: {}, restart it", old_pid);
-
     // kill it to restart
     let _ = Command::new("pkill")
         .arg("rsyslogd")
         .output()
-        .context("Failed to stop rsyslogd")?;
+        .context("Failed to restart rsyslogd")?;
```

Comment:
What would happen if rsyslogd wasn't running when pkill is called?

```
~> pkill doesnotexist
~> echo $?
1
```

Does it mean it would return an error and start_compute would fail?

Reply:
this error refers to `pkill` failure,
returning error is not a command failure